routes/index.js

```diff
@@ -8,14 +13,52 @@ router.get('/', function(req, res) {
     });
 });
 router.post('/pushData', function(req, res, next) {
+    console.log("starting");
     var db = req.db;
     var resultsHolder = db.get('formResults');
-    resultsHolder.insert(req.body, function(err, result) {
-        if (err)
-            res.send('Error');
-        else
-            res.send('Success');
-    });
+    if(currentEmail == ''){
+        res.send('No Email');
+    }else if(!(currentEmail.split('@')[1] == "nuevaschool.org")){
+        res.send('Invalid Email');
+    }else{
+        var error = false;
+        var contentObject = JSON.parse(req.body.content);
+        for(var storedData in contentObject){
+            if(req.body.content.hasOwnProperty(storedData)){
+                cachedData = JSON.parse(contentObject[storedData]);
+                resultsHolder.insert(cachedData, function(err, result) {
+                    if(err){
+                        console.log("Erroring with " + err);
+                        error = true;
+                    }
+                });
+            }else{
+                console.log("No own property for " + storedData);
+            }
+        }
+        if (error){
+            res.send('Data could not be entered.');
+        }else{
+            res.send('Completed');
```

Comment:
Please consider using HTTP status codes to differentiate between successful and failed requests.

Brief explanation and list of codes on MDN: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status